spec/validators/claim/advocate_hardship_claim_validator_spec.rb

```diff
@@ -274,181 +281,30 @@
+  
+    context 'when there is 1 basic fee uplift' do
+      before do
+        create(:basic_fee, :ndr_fee, claim: claim, quantity: 1, amount: 21.01)
       end
 
-      context 'with 2 defendants' do
-        before do
-          create(:defendant, claim: claim)
-          create(:misc_fee, fee_type: midtw, claim: claim, quantity: 1, amount: 21.01)
-          claim.reload
-        end
-
-        context 'when there are multiple uplifts of 1 per fee type' do
-          before do
-            create(:misc_fee, fee_type: miahu, claim: claim, quantity: 1, amount: 21.01)
-            create(:misc_fee, fee_type: midwu, claim: claim, quantity: 1, amount: 21.01)
-          end
-
-          it 'test setup' do
-            expect(claim.defendants.size).to eql 2
-            expect(claim.misc_fees.map { |f| f.fee_type.unique_code }.sort).to eql(%w[MIAHU MIAPH MIDTW MIDWU])
-          end
-
-          it 'should not error' do
-            should_not_error(claim, :base)
-          end
-        end
-
-        context 'when there are multiple uplifts of 2 (or more) per fee type' do
-          before do
-            create(:misc_fee, fee_type: miahu, claim: claim, quantity: 2, amount: 21.01)
-            create(:misc_fee, fee_type: midwu, claim: claim, quantity: 2, amount: 21.01)
-          end
-
-          it 'test setup' do
-            expect(claim.defendants.size).to eql 2
-            expect(claim.misc_fees.map { |f| f.fee_type.unique_code }.sort).to eql(%w[MIAHU MIAPH MIDTW MIDWU])
-          end
-
-          it 'should add one error only' do
-            should_error_with(claim, :base, 'defendant_uplifts_misc_fees_mismatch')
-            expect(claim.errors[:base].size).to eql 1
-          end
-        end
+      it 'test setup' do
+        expect(claim.defendants.size).to eql 1
+        expect(claim.basic_fees.map { |f| f.fee_type.unique_code }.sort).to include('BANDR')
       end
 
-      context 'when defendant uplifts fee marked for destruction' do
-        before do
-          create(:misc_fee, fee_type: miahu, claim: claim, quantity: 1, amount: 21.01)
-        end
-
-        it 'test setup' do
-          expect(claim.defendants.size).to eql 1
-          expect(claim.misc_fees.map { |f| f.fee_type.unique_code }.sort).to eql(%w[MIAHU MIAPH])
-          expect(claim).to be_invalid
-        end
-
-        it 'are ignored' do
-          miahu_fee = claim.fees.joins(:fee_type).where(fee_types: { unique_code: 'MIAHU' }).first
-          claim.update(
-            :misc_fees_attributes => {
-              '0' => {
-                'id' => miahu_fee.id,
-                '_destroy' => '1'
-              }
-            }
-          )
-          expect(claim).to be_valid
-        end
+      it 'should not error' do
+        should_not_error(claim, :base)
       end
 
-      context 'when defendants marked for destruction' do
+      context 'and form step is basic fees' do
```

Comment:
not a blocker general rule of thumb use `when`, `with`, `for` rather than `and` here

Reply:
👍  have amended to 'when form step is basic fees'